### ISSUE TITLE
Plots not generating

### ISSUE THREAD

**Issue by ans-qureshi** [tu-rbo/differentiable-particle-filters#3]
Hi,
I ran "simple.py" but the training doesn't end even when the best validation loss had not decreased for a while (screen-shot attached) and only the motion model shows up, the measurement model output and particle proposer don't show.
![validation loss error](https://user-images.githubusercontent.com/38855178/44844060-468a2b00-ac63-11e8-80c5-03e6a9dba6f2.png)

**Comment by ricoj** [comment]
Hi!

Our training method is quite patient and only stops training if the loss doesn't decrease for 200 epochs. In the your screen shot, you can see that the best epoch so far was epoch 1562. Training would stop in epoch 1762 if the loss does not decrease in the mean time, which would reset the patience counter to 200.

To speed up training (at the cost of performance), you can change the patience. The default value is defined [here](https://github.com/tu-rbo/differentiable-particle-filters/blob/69f55c260a480544d6f8e943412c10d4213802df/utils/exp_utils.py#L25
). I would recommend overwriting it [here](https://github.com/tu-rbo/differentiable-particle-filters/blob/69f55c260a480544d6f8e943412c10d4213802df/experiments/simple.py#L18) by inserting something like

    hyperparams['patience'] = 10


Cheers,
Rico

**Comment by ans-qureshi** [comment]
Got It! Thanks!
